files/image_config/platform/rc.local

```diff
@@ -240,6 +240,13 @@ if [ -f $FIRST_BOOT_FILE ]; then
     # Notify firstboot to Platform, to use it for reboot-cause
     touch /tmp/notify_firstboot_to_platform
 
+    # create /host/reboot-cause/platform/ directory
+    # /host/reboot-cause/platform/reboot-reason
+    # will be used to track last reboot reason
```

Comment:
This new 'platform' directory is not used by all platforms. I think this line should reflect that. Something like,

`# can be used to track last reboot reason by some platforms`

Reply:
Updated